docs/verify/simulate-with-qiskit-aer.ipynb

```diff
@@ -0,0 +1,255 @@
+   "source": [
+    "from qiskit.circuit.library import EfficientSU2\n",
+    "\n",
+    "n_qubits = 8\n",
+    "circuit = EfficientSU2(n_qubits)\n",
+    "circuit.decompose().draw(\"mpl\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This circuit contains parameters to represent the rotation angles for the $R_y$ and $R_z$. When simulating this circuit, we will need to specify explicit values for these parameters. In the next cell, we specify some values for these parameters and use the [Estimator](https://qiskit.org/ecosystem/aer/stubs/qiskit_aer.primitives.Estimator.html) primitive from Qiskit Aer to compute the exact expectation value of the observable $ZZ \\cdots Z$.\n",
+    "\n",
+    ">Note: Setting `approximation=True` when initializing the Estimator tells Qiskit Aer to approximate the effect of sampling error rather than actually perform sampling. This makes the simulation much more efficient, and also allows us to calculate the exact expectation value, free of sampling error. After Qiskit Aer 0.14, this will be the default behavior, so we won't need to specify this argument."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/tmp/ipykernel_64852/3684797947.py:8: DeprecationWarning: ``qiskit_aer.primitives.estimator.Estimator.__init__()``'s argument ``approximation`` is deprecated as of qiskit-aer 0.13. It will be removed no earlier than 3 months after the release date. approximation=True will be default in the future.\n",
+      "  exact_estimator = Estimator(approximation=True)\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "0.8870140234256602"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "from qiskit.quantum_info import SparsePauliOp\n",
+    "from qiskit_aer.primitives import Estimator\n",
+    "\n",
+    "observable = SparsePauliOp(\"Z\" * n_qubits)\n",
+    "params = [0.1] * circuit.num_parameters\n",
+    "\n",
+    "exact_estimator = Estimator(approximation=True)\n",
+    "job = exact_estimator.run(circuit, observable, params)\n",
+    "exact_value = job.result().values[0]\n",
+    "exact_value"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now, let's initialize a noise model that includes depolarizing error of 2% on every CX gate. In practice, the error arising from the two-qubit gates, which are CX gates here, are the dominant source of error when running a circuit. See [Building Noise Models](https://docs.quantum-computing.ibm.com/verify/building_noise_models) for an overview of constructing noise models in Qiskit Aer.\n",
+    "\n",
+    "In the next cell, we construct an Estimator that incorporates this noise model and use it to compute the expectation value of the observable."
```

Comment:
```suggestion
    "In the next cell, we construct an Estimator that incorporates this noise model and uses it to compute the expectation value of the observable."
```

Reply:
Your suggestion changes the meaning of the sentence. It is we who use the Estimator to compute, not the Estimator that uses the noise model to compute.

Reply:
Got it 👍